### PR TITLE
add ignore configuration

### PR DESCRIPTION
**Pull request by charlierudolph**
Used for `.gitignored` directories that aren't always present

A couple stylistic changes came in because of goimports. They will go away if #50 is merged

### CODE REVIEW

File: features/ignore_by_configuration.feature [hunk 1]
```diff
@@ -0,0 +1,96 @@
+Feature: ignore certain subprojects
+
+  As a developer
+  I want Morula to ignore folders that I gitignore
```
Comment (kevgo) [inline features/ignore_by_configuration.feature]: The examples below don't seem to use the gitignore file

**Comment by charlierudolph** [comment]
This [feature](https://github.com/Originate/morula/blob/a3700664d123171caee7e49d46b1cb4e9c4009a7/features/never.feature#L63) of `never` is annoying when the directory may or may not exist

File: features/ignore_by_configuration.feature (hunk 1 above)
Comment (charlierudolph) [inline features/ignore_by_configuration.feature]: Correct, this was meant to explain the use case.

**Comment by kevgo** [comment]
I would prefer to relax the `never` feature to allow listing non-existing directories than add another category for "don't look at these folders".

**Comment by charlierudolph** [comment]
Sounds good. I'll close this and open a PR for that.

**Pull request closed without merging**